Documentation~/Installation.md

```diff
@@ -1,6 +1,18 @@
 # Installation
 
-## Automatic Installation
+## Automatic Installation via DCC tools
+We can install everything through DCC tools. Currently, this is available only for Blender. For other DCC tools, see [Automatic Installation via Unity Editor](./Installation.md#automatic-installation-via-unity-editor).
```

Comment:
Can you please elaborate here what you mean by "everything" here ?

Reply:
Trying to convey both packages, MeshSync and the add-on zip. Might be better to phrase it as : "We can install MeshSync through DCC tools"?

Reply:
I made a few changes in general that should be more clear, let me know what you think